spruce-skill/eventsV2.md

```diff
@@ -0,0 +1,39 @@
+# Event Version 2
+
+The latest event version which supports Skill Views.
+
+## Event Object
+
+The incoming event is automatically processed and _might_ contain any (or none) of:
+
+* `userId` - The id of the User associated with this event
+
+* `locationId` - The id of the Location associated with this event
+
+* `organizationId` - The id of the Organization associated with this event
+
+If these ids are detected, a call is made to the Core API to get the data which are then attached to `ctx.auth`
```

Comment:
I think this needs to be reworded.   Maybe something like:

The Skill will automatically expand any of passed Id models and attach them to ctx.auth

Reply:
Updated - I removed the explanation of what was happening behind the scenes to keep it more straightforward